### PR TITLE
fix: reorganize endpoints

### PR DESCRIPTION
**Pull request by wa0x6e**
Reorganize API endpoints, for a consistent naming and path for all endpoints.

This service is getting more and more diverse features:
- #5 
- #9 
- #12 
- #15 

Reorganizing now to lay the ground

The /webhook has been isolated, as it'll be shared by all other futures update.

### CODE REVIEW

File: src/webhook.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+import log from './helpers/log';
+import { rpcError, rpcSuccess, storageEngine } from './helpers/utils';
+import VotesReport from './lib/votesReport';
+import { queues } from './lib/queue';
+
+const router = express.Router();
+
+router.post('/webhook', async (req, res) => {
+  log.info(`[http] POST /votes/generate`);
```
Comment (ChaituVR) [inline src/webhook.ts:10]:
```suggestion
  log.info(`[http] POST /webhook`);
```

File: README.md [hunk 2]
```diff
@@ -66,74 +64,84 @@ yarn test:e2e
...
 
-### Generate a cache file
+#### Generate a cache file
 
-Send a POST request with a body following the [Webhook event object](https://docs.snapshot.org/tools/webhooks).
+Send a `POST` request with a body following the [Webhook event object](https://docs.snapshot.org/tools/webhooks).
 
 ```bash
-curl -X POST localhost:3000/votes/generate \
+curl -X POST localhost:3000/webhook \
```
Comment (ChaituVR) [inline README.md:89]: Should be /api/webhook no?

File: src/webhook.ts
Comment (ChaituVR) [inline src/webhook.ts:1]: we are not using this file anywhere 🤔

File: README.md (hunk 2 above)
Comment (wa0x6e) [reply README.md:89]: I would like to keep /api only for output data, and /webhook outside it. wdyt ?

Comment (ChaituVR) [reply README.md:89]: makes sense as it is not really api

File: src/index.ts [hunk 3]
```diff
@@ -3,15 +3,27 @@ import express from 'express';
...
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 app.use(express.json({ limit: '4mb' }));
 app.use(cors({ maxAge: 86400 }));
 app.use(compression());
-app.use('/', api);
+app.use('/api', api);
+app.use('/', webhook);
```
Comment (ChaituVR) [inline src/index.ts:18]: I think it should be `/webhook` here and `/` in webhook.ts just to avoid confusion with `/` route below :)

Comment (wa0x6e) [reply src/index.ts:18]: then the `log.info(`[http] POST /webhook`);` inside webhook will not match the router ...mhhh

Comment (ChaituVR) [reply src/index.ts:18]: You think we can get more webhook routes in the future? if not we can just make in `app.post` here, no need of a separate router, if we think we may get more webhooks - then we should have a route like `webhook/events` so we can recognize them easily

Comment (wa0x6e) [reply src/index.ts:18]: file will grow for sure, at least 2 other PR are using it. But we will have only one single /webhook endpoint for all webhooks

**Review by ChaituVR — APPROVED** [review]
Tested Ack

**Pull request merged**